web/src/desktop_notifications.ts

```diff
@@ -0,0 +1,107 @@
+import $ from "jquery";
+
+import type {ElectronBridge} from "./electron_bridge";
+import type {Message} from "./message_store";
+
+const electron_bridge: ElectronBridge | undefined = window.electron_bridge;
```

Comment:
Extracting this variable is an unnecessary change.

Reply:
I did that because it was giving some error of 'possible undefined ... '. But I checked now and it is working perfectly without extracting this variable. I may have done some mistake before.

I will revert this change.